filters/gender_bias_filter/gender_bias_filter.py

```diff
@@ -0,0 +1,88 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class GenderBiasFilter(SentenceOperation):
+    tasks = [TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+
+    def __init__(self):
+        super().__init__()
+
+    @staticmethod
+    def flag_sentences(sentences):
+        flagged_sentences = []
+        feminine = ["she", "her", "hers"]
```

Comment:
Thanks for your submitted PR! However, I worry just counting pronouns can easily miss cases where e.g. human names are used. Have you considered including human names as well? For example, [this paper on gender bias](https://arxiv.org/pdf/2106.01601.pdf) relied on the popular human names collected in [checklist](https://github.com/marcotcr/checklist/blob/master/checklist/data/names.json).

Reply:
Hi, thanks for your comment and the links. I believe you were analyzing the very first version of the filter, which was indeed only based on pronouns counting. Currently there are such feminine/masculine attributes as relation, titles, jobs:
```  feminine = (
                ["she", "her", "hers"]
                + feminine_relation
                + feminine_relation_plural
                + feminine_titles
                + feminine_jobs
                + feminine_jobs_plural
            )
            masculine = (
                ["he", "him", "his"]
                + masculine_relation
                + masculine_relation_plural
                + masculine_titles
                + masculine_jobs
                + masculine_jobs_plural
            )``` 
and these are present in 4 languages (English, French, Polish, Russian). 

Since there is already a submission for the names filter, which is based on a large corpus of data, I thought it will be redundant to add names filtering to the gender bias filter, taking into account, that I left the possibility for the user to extend the feminine/masculine attribute lists with the input parameters.  

Let me know if you still believe that the change is required.

Reply:
I added a list of feminine and masculine names in four languages (English, French, Polish and Russian) to the GenderBiasFilter (the oe that this comment concerns).